docs/advanced-techniques.md

```diff
@@ -37,6 +37,147 @@ Resolve comes with a set of adapters covering popular DBMS choices. You can also
 
 Note that reSolve does not force you to use adapters. For example, you may need to implement a Read Model on top of some arbitrary system, such as a full-text-search engine, OLAP or a particular SQL database. In such case, you can just work with that system in the code of the projection function and query resolver, without writing a new Read Model adapter.
 
+## Custom Read Models
```

Comment:
I think we should emphasize that 'custom' here means 'custom storage'. Read model itself remans the same, but we store it in a different way, compared to storage options provided out-of -the-box (i.e. file instead of PostgreSQL or MySQL)

Reply:
Makes sense. I renamed the section to "Custom Read Model Connectors". Also, rewrote the intro like so: "You can implement a custom Read Model connector to define how a Read Model's data is stored. A connector implements the following functions:"